qa/tasks/mgr/dashboard/test_health.py

```diff
@@ -111,7 +114,8 @@ def test_full_health(self):
                     'total_bytes': int,
                     'total_objects': int,
                     'total_percent_used': float,
-                    'total_used_bytes': int
+                    'total_used_bytes': int,
+                    'total_used_bytes_raw': int
```

Comment:
might want to drop this line.

```
...
['stats']`: missing keys: set(['total_used_bytes_raw'])"
```

see http://pulpito.ceph.com/kchai-2018-12-05_13:56:41-rados-wip-kefu2-testing-2018-12-05-2007-distro-basic-smithi/3309107/